docs/shared/packages/linter/eslint-plugin-nx.md

```diff
@@ -1,4 +1,4 @@
-A plugin containing a collection of recommended ESLint rule configurations wrapped as ESLint plugins and an Nx specific [enforce-module-boundaries](#enforce-module-boundaries) rule.
+The `@nrwl/eslint-plugin-nx` plugin contains a collection of recommended ESLint rule configurations wrapped as ESLint plugins and an Nx-specific [enforce-module-boundaries rule](#enforce-module-boundaries-rule).
```

Comment:
The use of plugins here multiple times is currently inaccurate and confusing. I've added a suggestion here, but please note that this text is reference many times and it will need to be updated throughout the PR.

```suggestion
The `@nrwl/eslint-plugin-nx` package is an ESLint plugin which contains a collection of recommended ESLint rule configurations which you can extend from in your own ESLint configs, as well as an Nx-specific lint rule called [enforce-module-boundaries](#enforce-module-boundaries-rule).
```